frontend/src/utils/format.ts

```diff
@@ -104,6 +104,26 @@ export const convertTimestampToRelative = (
 		addSuffix,
 	})
 
+export const tillNextPayday = (timestamp: string, nextPaydayTime: string) => {
+	const time = parseISO(timestamp)
+	const paydayTime = parseISO(nextPaydayTime)
+	const nextDay = addDays(time, 1)
+
+	const nextDayPayDayTime = new Date(
+		Date.UTC(
+			nextDay.getUTCFullYear(),
+			nextDay.getUTCMonth(),
+			nextDay.getUTCDate(),
+			paydayTime.getUTCHours(),
+			paydayTime.getUTCMinutes(),
+			paydayTime.getUTCSeconds(),
+			paydayTime.getUTCMilliseconds()
+		)
```

Comment:
I'm not sure why this is correct. Suppose that `time` is on day X at 9:59. And paydays are each day at 10.

Then would this not return day `X+1` at 10? Why is this correct for the purposes of computing baker removal time?

I think you need something like
```
if time is before paydayTime on the same day then use paydayTime
otherwise use paydayTime + 1 day
```

Reply:
Perhaps I misunderstood the context.

Reply:
@abizjak - thanks its been fixed